themes/classic/modules/ps_emailsubscription/views/templates/hook/ps_emailsubscription-column.tpl

```diff
@@ -0,0 +1,64 @@
+{**
+ * 2007-2019 PrestaShop and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Academic Free License 3.0 (AFL-3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/AFL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/AFL-3.0 Academic Free License 3.0 (AFL-3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ *}
+
+<div class="block_newsletter">
+    <form action="{$urls.current_url}" method="post">
+        <div class="row">
+            <p id="block-newsletter-label" class="col-lg-12 col-md-12 col-xs-12">{l s='Get our latest news and special sales' d='Shop.Theme.Global'}</p>
+            <div class="col-lg-12 col-md-12 col-xs-12 mb-1">
+                <input
+                        class="btn btn-primary float-xs-right"
+                        name="submitNewsletter"
+                        type="submit"
+                        value="{l s='OK' d='Shop.Theme.Actions'}"
+                >
+                <div class="input-wrapper">
+                    <input
+                            name="email"
+                            type="email"
+                            value="{$value}"
+                            placeholder="{l s='Your email address' d='Shop.Forms.Labels'}"
+                            aria-labelledby="block-newsletter-label"
+                    >
+                </div>
+                <input type="hidden" name="action" value="0">
+                <div class="clearfix"></div>
+            </div>
+            <div class="col-lg-12 col-md-12 col-xs-12">
+                {if $conditions}
```

Comment:
Don't know if it's better, but to prevent undefined variable, we can't maybe add a check for existence?
```suggestion
                {if !empty($conditions)}
```

Reply:
:ok_hand: